templates/questions/index.html.ep

```diff
@@ -8,6 +8,12 @@
 	<% my $user = $c->model->oauth2->find($c->session('id')); %>
 </head>
 <title>Ask Me Anything</title>
+<script type="text/javascript">
+        if (window.location.protocol != "https:") {
+            window.location.href = "https:" + window.location.href.substring(window.location.protocol.length);
+        }
+</script>
+
```

Comment:
Would be great to make an independent library out of this, so that other apps could easily add this feature as well!!